paper/paper.md

```diff
@@ -0,0 +1,281 @@
+  connectome representation of the structural and functional brain systems, from any
+  sMRI / dMRI / rfMRI dataset structured according to the BIDS standard.
+It relies on Nipype [@GorgolewskiNipype:2011] and adopts a modular architecture.
+As illustrated by \autoref{fig:cmp3-diagram}, the workflow is composed of three different
+  pipelines (anatomical, diffusion, and fMRI) dedicated to the processing of each modality (sMRI, dMRI, rfMRI).
+
+![\textbf{Overview of the Connectome Mapper 3 BIDS App's workflow.}
+\label{fig:cmp3-diagram}](cmp3-diagram.png)
+
+Each pipeline is represented by a Nipype workflow that takes a BIDS formatted dataset as input, and
+  runs a number of sub-workflows (stages).
+Each stage implements one or multiple tasks, where each task can interface with either
+  a specific tool including in FSL [@Jenkinson2012FSL], FreeSurfer [@Fischl2012FreeSurfer],
+  ANTs [@AVANTS2008SymmetricBrain], dipy [@Garyfallidis2014DipyData],
+  mrtrix3 [@Tournier2019MRtrix3:Visualisation], AFNI [@Cox2012], or with an in-house tool
+  (\autoref{fig:cmp3-diagram}).
+
+To guarantee consistent processing in large neuroimaging cohorts,
+  pipeline and stage parameters can be set by loading pipeline
+  configuration files in `.json` format.
+Adopting BIDS allows CMP3 to automatically identify the structure of the input data, and to check the
+  availability of sMRI, dMRI, rfMRI, and derived data.
+Depending on the input data, the processing pipelines and stages are then dynamically built and configured
+  based on the parameters set in the different configuration files.
+Empowered by the Nipype workflow engine, the re-execution of the workflow will resume the
+  processing at any stages a change of parameter occurred.
+
+To ensure reproducibility and maximize re-usability of the tool, the implemented pipelines are encapsulated
+  in a Docker [@merkeldocker:2014] and a Singularity [@Kurtzer2017Singularity:Compute] software image
+  containers, in concordance to the BIDS App framework [@GorgolewskiBIDSMethods:2017].
+This means that the BIDS App of CMP3 can be run on a large diversity of datasets on Linux, MacOSX, Windows computers,
+  and on high performance computing systems (clusters) for large-scale analysis.
+
+Design considerations make CMP3 not only easy to install and use (as it provides the user with a computing
+  environment in which the pipelines are guaranteed to run, and where all dependencies
+  are already installed) on a diversity of multi-modal BIDS datasets, but also efficient in managing and
+  scaling the pipeline execution while recording provenance, and easy to customize and extend it for specific needs.
+At the time EEG is being integrated, CMP3 already provides a collection of interfaces dedicated
+  for this modality that would allow anyone to map the connectivity derived from EEG in
+  the CMP3 framework, as demonstrated in a tutorial notebook.
+
+\textbf{A revisited and extended multi-scale cortical parcellation scheme.}
```

Comment:
The Lausanne Parcelation is described in great detail while fMRI and dMRI pipline are barely mentioned. To not create too much work maybe add a little pargraph of 3 sentences that links to the relevant documentation: https://connectome-mapper-3.readthedocs.io/en/latest/bidsappmanager.html#diffusion-pipeline-stages
and
https://connectome-mapper-3.readthedocs.io/en/latest/bidsappmanager.html#fmri-pipeline-stages